### PR TITLE
Add ARM version of Plexamp for Apple Silicon

### PR DESCRIPTION
**Pull request by disrupted**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [ ] `brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

**Comment by disrupted** [comment]
@SMillerDev I've rebased manually now

File: Casks/plexamp.rb [hunk 1]
```diff
@@ -1,9 +1,16 @@
 cask "plexamp" do
   version "3.5.0"
-  sha256 "6eff736103de0ed89d81600f82e541deecc5f8d76d878320b41ad87ffe4bd87d"
 
-  url "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-#{version}.dmg",
-      verified: "plexamp.plex.tv/"
+  if Hardware::CPU.arm?
```
Comment (cho-m) [inline Casks/plexamp.rb:4]: All other casks with separate Intel/ARM branches use `if Hardware::CPU.intel?` for if-condition, so this should be changed to follow.

File: Casks/plexamp.rb [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
+    url "https://plexamp.plex.tv/plexamp.plex.tv/desktop/Plexamp-#{version}-arm64.dmg",
+        verified: "plexamp.plex.tv/"
+    sha256 "ead85dd518814718ce57b47a2cf76d0f290a9fd6465c88369d33964a67dbb76d"
```
Comment (cho-m) [inline Casks/plexamp.rb:7]: You should try to maintain the stanza order within the if-condition.
`sha256` should occur before `url`.
See https://docs.brew.sh/Cask-Cookbook#stanza-order

File: Casks/plexamp.rb (hunk 1 above)
Comment (disrupted) [reply Casks/plexamp.rb]: swapped it

**Review by cho-m — APPROVED** [review]
Thanks, @disrupted

**Pull request merged**